components/well-bold/well-bold-fr.html

```diff
@@ -51,7 +50,7 @@ <h3>Lien en vedette (Provisiore v1.0)</h3>
 
 <h3>Code</h3>
 <pre><code>
-&lt;section class="provisional well well-bold"&gt;
+&lt;section class="well well-bold"&gt;
```

Comment:
```suggestion
&lt;div class="well well-bold"&gt;
```
et met le code sample dans la documentation à jour aussi

Reply:
Changed.Thanks!